src/test/functions/env.spec.ts

```diff
@@ -135,26 +135,33 @@ BAR=bar
         want: { FOO: "foo" },
       },
       {
-        description: "should ignore comments",
+        description: "should handle empty values",
         input: `
-FOO=foo # comment
-# line comment 1
-# line comment 2
-BAR=bar # another comment
+FOO=
+BAR= "blah"
 `,
-        want: { FOO: "foo", BAR: "bar" },
-      },
-      {
-        description: "should ignore empty lines",
-        input: `
-FOO=foo
-
-
-BAR=bar
-
-`,
-        want: { FOO: "foo", BAR: "bar" },
+        want: { FOO: "", BAR: "blah" },
       },
+      //       {
+      //         description: "should ignore comments",
+      //         input: `
+      // FOO=foo # comment
+      // # line comment 1
+      // # line comment 2
+      // BAR=bar # another comment
+      // `,
+      //         want: { FOO: "foo", BAR: "bar" },
+      //       },
+      //       {
+      //         description: "should ignore empty lines",
+      //         input: `
+      // FOO=foo
+
+      // BAR=bar
+
+      // `,
+      //         want: { FOO: "foo", BAR: "bar" },
+      //       },
```

Comment:
Do we want to uncomment this?

Reply:
🤦  Forgot to save the file before commiting